### PR TITLE
Refactor code of main.go

### PR DESCRIPTION
**Pull request by Asphaltt**
Refactor code of attaching kprobe and attaching skb-tracking kprobe.

Signed-off-by: Leon Hwang <hffilwlqm@gmail.com>

### CODE REVIEW

**Comment by brb** [comment]
Hey @Asphaltt , is it ready for the review?

**Comment by Asphaltt** [comment]
Yes, it is ready.

File: internal/pwru/kprobe.go [hunk 1]
```diff
@@ -188,3 +197,49 @@ func AttachKprobeMulti(ctx context.Context, bar *pb.ProgressBar, kprobes []Kprob
 
 	return
 }
+
+func KprobeFuncs(ctx context.Context, funcs Funcs, coll *ebpf.Collection, a2n Addr2Name, useKprobeMulti bool, batch uint) *kprober {
```
Comment (brb) [inline internal/pwru/kprobe.go]: Nit: s/KprobeFuncs/NewKprober/

File: internal/pwru/kprobe.go [hunk 2]
```diff
@@ -243,3 +243,23 @@ func KprobeFuncs(ctx context.Context, funcs Funcs, coll *ebpf.Collection, a2n Ad
 
 	return &k
 }
+
+func KprobeNonSkbFuncs(nonSkbFuncs []string, funcs Funcs, coll *ebpf.Collection) *kprober {
```
Comment (brb) [inline internal/pwru/kprobe.go:247]: Nit: NewNonSkbFuncsKprober

**Review by brb — APPROVED** [review]
Thanks! Two nits (can be addressed in follow-up PRs).

File: internal/pwru/kprobe.go (hunk 1 above)
Comment (Asphaltt) [reply internal/pwru/kprobe.go]: Got it.

File: internal/pwru/kprobe.go (hunk 2 above)
Comment (Asphaltt) [reply internal/pwru/kprobe.go:247]: Got it.

**Pull request merged**